R/build-articles.R

```diff
@@ -86,13 +86,27 @@
 #' (If your package name includes a `.`, e.g. `pack.down`, use a `-` in the
 #' vignette name, e.g. `pack-down.Rmd`.)
 #'
-#' ## Missing topics
+#' ## Missing articles
 #'
 #' pkgdown will warn if there are (non-internal) articles that aren't listed
 #' in the articles index. You can suppress such warnings by listing the
 #' affected articles in a section with `title: internal` (case sensitive);
 #' this section will not be displayed on the index page.
+#' 
+#' ## External articles
```

Comment:
this section is incomplete.

adding the `external-articles:` entry isn't sufficient, and should be accompanied by adding the `name:`s to the `contents:` of `articles:`